simulator.go

```diff
@@ -59,6 +68,16 @@ func startSimulator() error {
 	}
 	defer shutdownFn3()
 
+	// If ExternalImportEnabled is enabled, the simulator import resources
+	// from the existing cluster that indicated by the `KUBECONFIG`.
+	if cfg.ExternalImportEnabled {
+		ctx := context.Background()
+		// This must be called after `StartScheduler`
+		if err := dic.ReplicateExistingClusterService().ImportFromExistingCluster(ctx); err != nil {
+			return xerrors.Errorf("import existing cluster: %w", err)
+		}
+	}
+
```

Comment:
Move this logic to L62 (before starting simulator server)
If the server is started before the import is finished, you will see an empty resource list, depending on the timing of the front-end startup.